src/npm-fastui-bootstrap/src/index.tsx

```diff
@@ -75,6 +75,7 @@ export const classNameGenerator: ClassNameGenerator = ({
     case 'FormFieldBoolean':
     case 'FormFieldSelect':
     case 'FormFieldSelectSearch':
+    case 'FormFieldRadio':
```

Comment:
looking at your screenshot vs [the bootstrap docs for Radios](https://getbootstrap.com/docs/5.0/forms/checks-radios/#radios) it appears you're missing some styling.

I think you need ot add some more classes here for radios, perhaps `form-check`, `form-check-input` and `form-check-label`?

Reply:
I added both `form-check-input` and `form-check-label` - but i am unsure where i should add the class `form-check`, i tried a few different places but it never seemed to work as intended.
as it stands, here is how it looks:
![image](https://github.com/pydantic/FastUI/assets/86600518/baa765fd-e7a4-4abe-8bb5-a58d929cbb30)